### ISSUE TITLE
Who else thinks they're screwed for PA2

### ISSUE THREAD

**Issue by lewistham9x** [nus-cs2030/2122-s1#251]

**Comment by ryanwalterlee** [comment]
class me {
    amScrewedForPA2() {
        return true;
    }
}

**Comment by JavierTham** [comment]
Don't worry! For me, only my toString method was on the right track :rofl:

**Comment by kavxya** [comment]
oh wells the end is near :'))

**Comment by RussellDash332** [comment]
> Don't worry! For me, only my toString method was on the right track 🤣

Imagine forgetting this during PA2 😢
But for real, I almost did.

**Comment by Devvynneo** [comment]
Just checking, 
is it normal to not understand the sample solution given? 
Because I find the practical hard to follow even after the solution

**Comment by huzaifa1712** [comment]
I definitely am, had to change pretty much every line. I think a lot of people found it difficult though, just from talking to classmates and what my TA said.

**Comment by jehousoh** [comment]
it's okay most of my friends including myself had to change the majority of our code so dont worry too much ig HAHA :|

**Comment by josiahoo** [comment]
Same here, most of the people I've talked to found that the sample solution was really different from their own implementations (including me too hahaha). Just try your best to modify our code as sparingly as possible, and don't beat yourself up if you have to change a lot because many people really struggled with this PA, be happy that you did the best you could in the 1.5hrs. Atb for finals and the project(s) too!!

**Comment by bingsen0806** [comment]
> Don't worry! For me, only my toString method was on the right track 🤣

I realized after PA2 that I wrote "Random" instead of "Rand" in my toString () lol. Now even my toString() needs to be changed T_T

**Comment by Richeton** [comment]
Same sentiments w everything being changed (and not fully understanding what's going on/zoning out during the PA2)... Hopefully, we can end up pulling through this final stretch... :') Jiayous, everyone!

**Comment by CedricPei** [comment]
The same thing that the only unchanged method is toString() method......hope at least I can get 1 mark.

**Comment by yuehernkang** [comment]
Changed many things too ! But hope it goes well

**Comment by satsuma42** [comment]
> > Don't worry! For me, only my toString method was on the right track 🤣
> 
> Imagine forgetting this during PA2 😢 But for real, I almost did.

I actually did forget this like an idiot, but wrote a placeholder. Good thing there isn't much to it anyway.

**Comment by teominghuang** [comment]
Same, I guess we can only pray :(.

**Comment by Trooksy** [comment]
Yeah looking at the current code it is very different from what I did in the exam
